src/test/java/ReverserTest.java

```diff
@@ -0,0 +1,54 @@
+import org.junit.Test;
+import static org.junit.Assert.*;
+import org.junit.Ignore;
+
+public class ReverserTest {
+    @Test
+    public void reverse_ShouldReversWholeWord_ifOneWordAndOnlyCapitalLetters() {
+        String str = "ABC";
+        String ans = "CBA";
+        String expect = ans;
+        String result = Reverser.reverseWithoutSpecialSymbols(str);
+        assertEquals(expect,result);
+    }
+    @Test
+    public void empty_ShouldThrowAnExceptionIfStringEmpty() {
+        String str = "ABC";
+        String ans = "";
+        String expect = ans;
+        String result = Reverser.reverseWithoutSpecialSymbols(str);
+        assertNotEquals(expect,result);
```

Comment:
Please read some info about checking exceptions in JUnit4. Here you check equality, but you should check the throwing of exception.